randovania/games/dread/logic_database/Burenia.txt

```diff
@@ -2091,16 +2091,16 @@ Extra - asset_id: collision_camera_019
               Spider Magnet or Grapple Movement (Beginner)
           Flash Shift and Wall Jump (Beginner)
   > Dock to Early Gravity Speedboost Room 1 (Lower)
-      After Burenia - Early Gravity Speed Blocks Destroyed and Speed Booster Conservation (Beginner)
+      After Burenia - Early Gravity Speed Blocks Destroyed
   > Water Pit
       Trivial
   > Event - Early Gravity Speed Blocks Destroyed
-      Speed Booster and After Burenia - Ghavoran Teleoport Speed Blocks Destroyed
+      Speed Booster and After Burenia - Ghavoran Teleoport Speed Blocks Destroyed and Speed Booster Conservation (Beginner)
 
 > Event - Early Gravity Speed Blocks Destroyed; Heals? False
   * Layers: default
   * Event Burenia - Early Gravity Speed Blocks Destroyed
-  > Dock to Early Gravity Speedboost Room 1 (Lower)
+  > Slope Top
```

Comment:
My knowledge isnt the best right now, but dont you need some upgrades to jump back up? On the db map this seems to be too high to be a trivial jump.

Reply:
We have the easiest wall jumps as trivial, since the game's own hints explain them.